3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb

```diff
@@ -171,17 +184,32 @@
    "metadata": {},
    "source": [
     "## Setup transforms for training and validation\n",
-    "To save on GPU memory utilization, the num_samples can be reduced to 2. "
+    "To save on GPU memory utilization, the num_samples can be reduced to 2. \n",
+    "\n",
+    "A note on design related to MetaTensors:\n",
+    "\n",
+    "- Summary: using `EnsureTyped(..., track_meta=False)` and `set_track_meta(False)` (later) speeds up training significantly.\n",
+    "\n",
+    "- We are moving towards the use of MONAI's MetaTensor in place of numpy arrays or PyTorch tensors. MetaTensors have the benefit of carrying the metadata directly with the tensor, but in some use cases (like here with training, where training data are only used for computing loss and metadata is not useful), we can safely disregard the metadata to improve speed.\n",
+    "\n",
+    "- Hence, you will see `EnsureTyped` being used before the first random transform in the training transform chain, which caches the result of deterministic transforms on GPU as Tensors (rather than MetaTensors), with `track_meta = False`. \n",
+    "\n",
+    "- On the other hand, in the following demos we will display example validation images, which uses metadata, so we use `EnsureTyped` with `track_meta = True`. Since there are no random transforms during validation, tracking metadata for validation images causes virtually no slowdown (~0.5%).\n",
```

Comment:
Also need to update this description for the validation EnsureTyped transform.

Thanks.

Reply:
Same as above.

Thanks.